crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -1920,6 +1921,74 @@ pub enum SuiEvent {
     Checkpoint(CheckpointSequenceNumber),
 }
 
+impl TryFrom<SuiEvent> for Event {
+    type Error = anyhow::Error;
+    fn try_from(event: SuiEvent) -> Result<Self, Self::Error> {
+        Ok(match event {
+            SuiEvent::MoveEvent {
+                package_id,
+                transaction_module,
+                sender,
+                type_,
+                fields: _,
```

Comment:
I'm disliking this field more and more over time. Do we know if community is using this field?

Reply:
yes people are using it, I will be revisiting this sometime soon, I will be adding a DataEncoding param to let user choose between BCS and parsedJSON, to avoid an optional field.

Reply:
Fields is put there originally to capture all the fields from Move events, and to allow for us to filter any field.  I still think this would be valuable, but if nobody uses it we can get rid of it.   The BCS field is not queryable by any database.

Fields is intended to be returned as JSON actually.